qcelemental/models/basemodels.py

```diff
@@ -127,6 +127,7 @@ def serialize(
         exclude_unset: Optional[bool] = None,
         exclude_defaults: Optional[bool] = None,
         exclude_none: Optional[bool] = None,
+        **kwargs,
```

Comment:
My only hesitation is because I don't know if serialize was written w/o kwargs as a defensive measure. Certainly the indentation use to which you've put kwargs is reasonable and harmless, though. @bennybp, any complications to the database if formats can change through this?

Reply:
I don't believe this will cause any problems on my end